extensions/amp-story/1.0/pagination-buttons.js

```diff
@@ -222,50 +208,42 @@ export class PaginationButtons {
   onCurrentPageIndexUpdate_(pageIndex) {
     const totalPages = this.storeService_.get(StateProperty.PAGE_IDS).length;
 
-    if (pageIndex === 0) {
-      this.backButton_.updateState(BackButtonStates.HIDDEN);
-    }
-
-    if (pageIndex > 0) {
-      this.backButton_.updateState(BackButtonStates.PREVIOUS_PAGE);
-    }
+    // Hide back button if no previous page.
+    this.setEnabled_(this.backButton_, pageIndex > 0);
 
     if (pageIndex < totalPages - 1) {
-      this.forwardButton_.updateState(ForwardButtonStates.NEXT_PAGE);
+      this.forwardButton_.updateState(ButtonStates.NEXT_PAGE);
     }
 
     if (pageIndex === totalPages - 1) {
```

Comment:
Nit: This could be an `else` right? (unrelated to the PR, but makes it explicit that these two conditions are non-overlapping)